docs/Makefile

```diff
@@ -16,7 +16,7 @@ clean:
 	rm -rf $(BUILDDIR)/*
 
 deps:
-	pip install --disable-pip-version-check -r requirements.txt
```

Comment:
Oh I see. Now I'm torn between improving consistency and reducing pointless log messages...

Meh, this is trivial either way, LGTM.